src/SamplesApp/UITests.Shared/Windows_Phone/Devices_Notifications_VibrationDevice.xaml.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Uno.UI.Samples.Controls;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.Foundation.Metadata;
+using Windows.Phone.Devices.Notification;
```

Comment:
I think you should use the _new_ version of UWP in the namespace `Windows.Devices.Haptics` instead of this one
<https://docs.microsoft.com/en-us/uwp/api/windows.devices.haptics.vibrationdevice>.

Reply:
Seems like a very different API, can't even find any good samples for it 🤔 . But even then - we should implement this one as well anyway, or not?

Reply:
Good question. I think the one coming in the "Phone SDK" is the old one and the idea is to replace it by the new APIs. I think the _phone_ version will only work on a _Windows Phone_ device but the newer API should work everywhere (even if a vibration device is uncommon on a non-phone device).

There's a Microsoft sample : <https://github.com/Microsoft/Windows-iotcore-samples/tree/develop/Samples/VibrationDevice>

My suggestion would be to implement both in Uno to have a full coverage.

For the building problem, @jeromelaban told me we can install it like what is done in the WindowsCommunityToolkit:
  1. Instruction is the `.vsts-ci.yaml` file: <https://github.com/nventive/Uno.WindowsCommunityToolkit/blob/6c0bc8a80144b700fc96dacd5db3f45c29d2b3a0/.vsts-ci.yml#L42>
  2. The powershell to download the SDK ISO, mount it and install missing extension SDK: <https://github.com/nventive/Uno.WindowsCommunityToolkit/blob/6c0bc8a80144b700fc96dacd5db3f45c29d2b3a0/build/Install-WindowsSdkISO.ps1>

Reply:
So I would suggest we close merge this PR (once the build is fixed) and then I will look into the new vibration API separately in a new PR.

But for the build - could I ask for your assistance? I have no experience with YAML yet so it would be quite painful :-D .

Reply:
Hi, @carldebilly , @jeromelaban , can you please help with the build/YAML change? I am really unsure what to do to make it work properly 😅 .

Reply:
You need to add:
```
  - powershell: .\build\Install-WindowsSdkISO.ps1 17763
    displayName: Insider SDK
```
to the yaml for the windows build (after the NodeTool@0 step), and add the `Install-WindowsSdkISO.ps1` to the build directory. That should be enough :)